app/src/main/res/values-pt/strings.xml

```diff
@@ -279,7 +279,7 @@
         <b>PROTECT &amp; HIDE PHOTOS, VIDEOS &amp; FILES</b>
         Using pin, pattern or your device’s fingerprint scanner you can protect and hide photos, videos &amp; entire albums. You can protect the app itself or place locks on specific functions of the app. For example, you can’t delete a file without a fingerprint scan, helping to protect your files from accidental deletion.
 
-        <b>Check out the full suite of Simple Tools here:</b>
+        <b>Consulte todas as aplicações Simple Tools aqui:</b>
```

Comment:
basically the whole long app description is untranslated, why have you translated just this line? :)